shell/platform/tizen/channels/text_input_channel.cc

```diff
@@ -425,13 +425,13 @@ bool TextInputChannel::FilterEvent(Ecore_Event_Key* event) {
   imf_event.dev_name = is_ime ? "ime" : "";
   imf_event.keycode = event->keycode;
 
+#ifdef WEARABLE_PROFILE
+  // FIXME: Only for wearable.
   if (is_ime && strcmp(event->key, "Select") == 0) {
-    if (engine_->device_profile == DeviceProfile::kWearable) {
-      // FIXME: for wearable
-      in_select_mode_ = true;
-      FT_LOGI("Set select mode[true]");
-    }
+    in_select_mode_ = true;
```

Comment:
Could you please completely exclude ```in_select_mode_``` on other profiles?

Reply:
Please review again.